isb_web/main.py

```diff
@@ -148,6 +150,61 @@ async def get_thing(
         content=content, media_type=item.resolved_media_type
     )
 
```

Comment:
I went back and forth on whether to add two methods or one with a parameter here.  I settled on two methods because even though the inputs are identical, the outputs are different kinds of objects.  I could be persuaded to go the other way, though.  What do you think @datadavev?

Reply:
TODO: cleanup the documentation